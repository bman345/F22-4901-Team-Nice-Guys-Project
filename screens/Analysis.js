import { StatusBar } from 'expo-status-bar';
import { StyleSheet, Text, View } from 'react-native';

export default function Analysis() {
  return (
    <View style={styles.container}>
      <Text>Hello World</Text>
      <Text>My name is Guest</Text>
      <StatusBar style="auto" />
    </View>
  );
}

const styles = StyleSheet.create({
  container: {
    flex: 1,
    backgroundColor: '#4628',
    alignItems: 'center',
    justifyContent: 'center',
  },
});
