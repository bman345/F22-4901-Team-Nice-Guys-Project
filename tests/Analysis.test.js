import React from 'react';
import renderer from 'react-test-renderer';
import Analysis from '../screens/Analysis';


test('renders correctly', () => {
    const tree = renderer.create(<Analysis />).toJSON();
    expect(tree).toMatchSnapshot();
  });