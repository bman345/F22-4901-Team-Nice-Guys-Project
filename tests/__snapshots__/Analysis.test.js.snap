// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<View
  style={
    Object {
      "alignItems": "center",
      "backgroundColor": "#4628",
      "flex": 1,
      "justifyContent": "center",
    }
  }
>
  <Text>
    Hello World
  </Text>
  <Text>
    My name is Guest
  </Text>
</View>
`;
